listenbrainz/webserver/static/js/src/utils.tsx

```diff
@@ -363,6 +364,12 @@ const getListenablePin = (pinnedRecording: PinnedRecording): Listen => {
   return pinnedRecListen;
 };
 
+const countWords = (str: string): number => {
```

Comment:
That's elegant!
If it comes from the internet, do put a link to the source as a comment